docs/operations/hotplug.md

```diff
@@ -0,0 +1,129 @@
+
+```yaml
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+      - HotplugVolumes
+```
+
+Once the feature gate is enabled, you will be able to hotplug volumes into a running VMI.
+
+## Virtctl support
+
+In order to hotplug a volume, you must first prepare a volume. This can be done by using a DataVolume (DV). In the example we will use a blank DV in order to add some extra storage to a running VMI
+
+```yaml
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: example-volume-hotplug
+spec:
+  source:
+    blank: {}
+  pvc:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 5Gi
+```
+In this example we are using ReadWriteOnce accessMode, and the default FileSystem volume mode. Volume hotplugging supports all combinations of block volume mode and ReadWriteMany/ReadWriteOnce/ReadOnlyMany accessModes, if your storage supports the combination.
+
+### Addvolume
+
+Now lets assume we have started a VMI like the [Fedora VMI in examples](examples/vmi-fedora.yaml) and the name of the VMI is 'vmi-fedora' we can add the above blank volume to this running VMI by using the 'addvolume' command  available with virtctl
+
+```bash
+$ virtctl addvolume vmi-fedora --volume-name=example-volume-hotplug
+```
+
+This will hotplug the volume into the running VMI, and set the serial of the disk to the volume name. In this example it is set to example-hotplug-volume.
+
+#### Serial
+You can change the serial of the disk by specifying the --serial parameter, for example:
+```bash
+$ virtctl addvolume vmi-fedora --volume-name=example-volume-hotplug --serial=1234567890
+```
+
+The serial will be used in the guest so you can identify the disk inside the by the serial. For instance in Fedora the disk by id will contain the serial
+```bash
+$ virtctl console vmi-fedora
+
+Fedora 32 (Cloud Edition)
+Kernel 5.6.6-300.fc32.x86_64 on an x86_64 (ttyS0)
+
+SSH host key: SHA256:c8ik1A9F4E7AxVrd6eE3vMNOcMcp6qBxsf8K30oC/C8 (ECDSA)
+SSH host key: SHA256:fOAKptNAH2NWGo2XhkaEtFHvOMfypv2t6KIPANev090 (ED25519)
+eth0: 10.244.196.144 fe80::d8b7:51ff:fec4:7099
+vmi-fedora login:fedora
+Password:fedora
+[fedora@vmi-fedora ~]$ ls /dev/disk/by-id
+scsi-0QEMU_QEMU_HARDDISK_1234567890
+[fedora@vmi-fedora ~]$ 
+```
+As you can see the serial is part of the disk name, so you can uniquely identify it.
+
+### Persist
+In some cases you want a hotplugged volume to become part of the standard disks after a restart of the VM. For instance if you added some permanent storage to the VM. We also assume that the running VMI has a matching VM that defines it specification. You can call the addvolume command with the --persist flag. This will update the VM domain disks section in addition to updating the VMI domain disks. This means that when you restart the VM, the disk is already defined in the VM, and thus in the new VMI.
```

Comment:
```suggestion
In some cases you want a hotplugged volume to become part of the standard disks after a restart of the VM. 
For instance if you added some permanent storage to the VM. We also assume that the running VMI has a matching VM that defines it specification. 
You can call the addvolume command with the `--persist` flag. This will update the VM domain disks section in addition to updating the VMI domain disks. 
This means that when you restart the VM, the disk is already defined in the VM, and thus in the new VMI.
```

Reply:
Personally not a big fan of hard line breaks like that. I prefer the renderer to break things up for me. but changed.